source/reference/options.txt

```diff
@@ -452,8 +452,10 @@ Session Options
 
 .. option:: --retryWrites
 
-   Enables :manual:`retryable writes </core/retryable-writes/>` as the
-   default for sessions in the |mdb-shell|.
+   Enables :ref:`retryable-writes`. Retryable writes are the default in
```

Comment:
Same suggestion from above - how about this to be a bit more transparent that the default is `on/true`?
```suggestion
   Enables :ref:`retryable-writes`. Retryable writes are enabled by default in
```

Reply:
Done